playbooks/acs.yml

```diff
@@ -25,12 +25,15 @@
     secret_ca_passphrase: "{{ hostvars.localhost.ca_signing_key_passphrase }}"
   ansible.builtin.import_playbook: pki.yml
 
-- name: Gather facts from all hosts
-  hosts: all:!external
+- name: Setup common facts
+  hosts: repository:identity
```

Comment:
That looks brutal. I don't think hosts without repo or identoity roles can complete without the common role being executed on them.
What theretional behind this change?

Reply:
common role is getting applied everywhere since it's a dependency of any other role, but the problem here is that I want to reference facts before the execution of any role, hence the explicit dependency requirement